backend/webscraper/output.json

```diff
@@ -0,0 +1,130 @@
+[
+    {
+        "name": "Breakfast",
+        "dining_halls": [
+            {
+                "name": "USC Village Dining Hall",
+                "stations": [
+                    {
```

Comment:
Since this is a temporary file, can we keep it out of the repo? We can do this by:

- Deleting the file
- Prevent it from being added the next time someone runs the tests by adding it to the .gitignore file (located in the backend directory, just add a line with `webscraper/output.json`

Reply:
I made a delete file function that deletes output.json after every test